Towards A Social Platform.md

```diff
@@ -0,0 +1,88 @@
+# Long term goals for Social Platform
+
+Ultimately we are building towards incorporation of a social platform into KBase. 
+
+The goals of the social platform are to:
+
+1. Provide the means for users to organize and share their data and results in the system. 
+2. Provide a means for individual labs to organize their data and analyses in a way that is persistently available to the lab lead. 
```

Comment:
I don't see how this is separable from the goals of KBase in general w.r.t. data. "Be consistent with goals for data in KBase" seems sufficient (with these as sub-bullets if you don't trust that people know what these are)